.pre-commit-hooks.yaml

```
@@ -0,0 +1,10 @@
+- id: jupytext
+  name: jupytext
+  description: Convert all notebooks into another format
+  language: python
+  entry: jupytext --add-untracked
+  pass_filenames: true
+  files: ipynb
```

Instead of files why not use `types: [jupyter]` like nbstripout uses?

---

It didn't work on my machine, probably I had an older pre-commit installed but I wasn't sure when it was added so I did it this way. But we will remove this and just pass all files, ignoring the ones that we aren't trying to process.

---

@JohnPaton, yeah I've run into this before. You need to run `pip install -U identify`. Annoying that you can specify a min version of identify in the hook (you can for pre-commit though), but it is the recommended way to do it and will make it a lot easier for end users to override the behavior and support other types.
```suggestion
  types: [notebook, python]
```
We can also add markdown, etc to the list of types: